### PR TITLE
Cypress org admin

### PR DESCRIPTION
**Pull request by nirajCITZ**
Added scenarios for JWK Public Key and CORS plugins

### CODE REVIEW

File: e2e/cypress/pageObjects/namespaceAccess.ts [hunk 1]
```diff
@@ -86,6 +86,34 @@ class NamespaceAccessPage {
...
+    cy.get('[role="table"]').should('be.visible').then(() => {
+      debugger
+      let textAccessUserName: string
+      cy.wait(2000)
+      let orgAdmins: Array<string> = members
+      debugger
+      orgAdmins.forEach(function (orgAdmin) {
+        debugger
+        cy.get('[role="table"]').find('tr').find('td:nth-child(2)').find('li').filter(':visible').each(($el) => {
+          debugger
```
Comment (jTendeck) [inline e2e/cypress/pageObjects/namespaceAccess.ts:101]: Few debugger statements here

File: e2e/cypress/pageObjects/webApp.ts [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+  }
+
+  getStatusAfterClickOnCORS(): Cypress.Chainable<any> {
+    cy.wait(5000)
+    cy.get(this.corsButton).click({force:true})
+    cy.wait(8000)
+    // Perform interactions or assertions on the page elements
+    debugger
+    return cy.get(this.corsResponseTxt).invoke('text').then((text:any) => {
+      debugger
```
Comment (jTendeck) [inline e2e/cypress/pageObjects/webApp.ts:28]: Debugger statements

**Review by Jonesy — APPROVED** [review]

File: e2e/cypress/tests/16-aps-api/05-authorizationProfiles.cy.ts
Comment (jTendeck) [inline e2e/cypress/tests/16-aps-api/05-authorizationProfiles.cy.ts:1]: Some debugger statements here as well

File: e2e/cypress/tests/17-CORS/01-create-api.cy.ts
Comment (jTendeck) [inline e2e/cypress/tests/17-CORS/01-create-api.cy.ts:1]: Here too

**Review by jTendeck — APPROVED** [review]

**Pull request merged**